src/libexpr/eval.hh

```diff
@@ -245,6 +245,16 @@ public:
     /** `"unknown"` */
     Value vStringUnknown;
 
+    using StorePathAccessors = std::map<CanonPath, ref<SourceAccessor>>;
```

Comment:
I think it would be more future-compat if this did not use `CanonPath` keys, both for lazy trees and for Windows.

Reply:
Right now it must use `CanonPath`, because we don't have a concept of non-`CanonPath`s in the virtual filesystem exposed in the evaluator.